### PR TITLE
API

### PR DESCRIPTION
**Pull request by sntgopl**
In this milestone I did:
- Created a game in the A.P.I. server.
- Fetch data from the server and use the refresh button to reload the list.
- Add a User and Score to the list and posting in the A.P.I. server.
- Changed functions to implement A.P.I. response type.

### CODE REVIEW

File: src/modules/post-api.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//     body: JSON.stringify({
+//       user: name,
+//       score,
+//     }),
+//   });
+//   const data = await response.json();
+//   return data;
+// }
+
+// export default sendData;
```
Comment (t-yanick) [inline src/modules/post-api.js]: - [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.

File: src/index.js [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
-import writeList from './modules/innerhtml.js';
 import Leader from './modules/leaders.js';
+import refreshList from './modules/refresh-api.js';
+import addLeader from './modules/post-api.js';
 import './style.css';
 
-writeList();
+// setInterval(() => {
+//   console.log(leaderboard[0][0]);
+// },3000);
```
Comment (t-yanick) [inline src/index.js]: - [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.

File: src/index.js [hunk 3]
```diff
@@ -1,26 +1,25 @@
-import { leaderboard, UpdateLocalStorage } from './modules/data.js';
+// import { leaderboard } from './modules/data.js';
```
Comment (t-yanick) [inline src/index.js]: - [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @sntgopl, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
Hi @sntgopl , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
goodjob

## Highlights
✔️ You're code is clean. well done 
✔️ your linters are all ok

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**